.github/workflows/lint-pr.yml

```
@@ -0,0 +1,15 @@
+---
+name: Lint PR
+on:
```

Either empty line or not in both files :D

```suggestion

on:
```